src/world.jl

```diff
@@ -563,14 +564,28 @@ end
 end
 
 """
-    @copy_entity!(world::World, entity::Entity; add::Tuple, remove::Tuple)
+    @copy_entity!(
+        world::World,
+        entity::Entity;
+        add::Tuple=(),
+        remove::Tuple=(),
+        copy=:copy,
```

Comment:
```suggestion
        mode=:copy,
```

Maybe mode could be better